### PR TITLE
Show gmos fpu

### PR DESCRIPTION
**Pull request by cquiroz**
Now with PosAngle we can display the fpu if selected

https://user-images.githubusercontent.com/3615303/166562678-9a86b998-a15b-4717-a0e6-c0ee39740a1e.mp4

### CODE REVIEW

File: model/shared/src/main/scala/explore/model/ObsSummary.scala [hunk 1]
```diff
@@ -131,7 +131,17 @@ object ObsSummaryWithTitleConstraintsAndConf {
...
-    Eq.by(o => (o.id, o.title, o.subtitle, o.constraints, o.status, o.activeStatus, o.duration))
+    Eq.by(o =>
+      (o.id,
+       o.title,
+       o.subtitle,
+       o.constraints,
+       o.status,
+       o.activeStatus,
+       o.duration,
+       o.scienceMode
```
Comment (toddburnside) [inline model/shared/src/main/scala/explore/model/ObsSummary.scala:142]: Do you know if this `scienceMode` is used elswewhere? Or is it only for the purpose of sending to the AladinContainer?

**Review by toddburnside — APPROVED** [review]
Looks good for Andy to check out. I'll try to fix the API/local sync problem in a PR tomorrow.

File: model/shared/src/main/scala/explore/model/ObsSummary.scala (hunk 1 above)
Comment (cquiroz) [reply model/shared/src/main/scala/explore/model/ObsSummary.scala:142]: It is (or will be used for step generation)

Comment (toddburnside) [reply model/shared/src/main/scala/explore/model/ObsSummary.scala:142]: The value in the ObsSummary will be used? It's also in the ObsEditQuery.

**Pull request merged**